File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AnimatedCuirass.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.HeavyArmor;

/* JADX INFO: loaded from: classes3.dex */
public class AnimatedCuirass extends HeavyArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_heavy_animated_cuirass_name;
        this.idDescription = R.string.armor_heavy_animated_cuirass_description;
        this.idImage = R.drawable.animated_cuirass;
        this.price = 756L;
        this.maxHp = 300;
        this.constitution = 10;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AnimatedCuirass.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.HeavyArmor

class AnimatedCuirass : HeavyArmor() {
    override fun configureProperties() {
        idName = R.string.armor_heavy_animated_cuirass_name
        idDescription = R.string.armor_heavy_animated_cuirass_description
        idImage = R.drawable.animated_cuirass
        price = 756L
        maxHp = 300
        constitution = 10
    }
}
